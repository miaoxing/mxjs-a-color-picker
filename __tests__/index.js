import 'core-js/stable';
import 'regenerator-runtime/runtime';

import React from 'react';
import ColorPicker from '..';
import {render} from '@testing-library/react';

describe('color picker', () => {
  test('basic', () => {
    const result = render(<ColorPicker/>);
    expect(result.container.innerHTML).toMatchSnapshot();
  });

  test('value', () => {
    const result = render(<ColorPicker value="red"/>);
    expect(result.container.innerHTML).toContain('background: red;');
  });
});
