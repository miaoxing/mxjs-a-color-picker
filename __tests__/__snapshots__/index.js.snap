// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color picker basic 1`] = `"<div><div style=\\"width: 60px; padding: 5px; background: rgb(255, 255, 255); border-radius: 1px; box-shadow: 0 0 0 1px rgba(0,0,0,.1); display: inline-block; cursor: pointer;\\"><div style=\\"height: 22px; border-radius: 2px; background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAMUlEQVQ4T2NkYGAQYcAP3uCTZhw1gGGYhAGBZIA/nYDCgBDAm9BGDWAAJyRCgLaBCAAgXwixzAS0pgAAAABJRU5ErkJggg==);\\"></div></div></div>"`;
